geojson/base.py

```diff
@@ -112,11 +112,6 @@ def to_instance(cls, ob, default=None, strict=False):
                     raise AttributeError(
                         "{0} is not a GeoJSON type").format(type_)
                 geojson_factory = getattr(geojson.factory, type_)
-                if not issubclass(geojson_factory, GeoJSON):
```

Comment:
to better understand what this removal, what makes you think this code is unreachable?

Reply:
Happy to clarify: it is unreachable because all the class objects that are attributes of `geojson.factory` are also subclasses of `GeoJSON`. If you want to ensure that this invariant *remains* true, you could add a test for it (or even put the validation directly in the `geojson.factory` module.

`geojson.factory` does have attributes that are not classes, but those throw an exception when passed into `issubclass` -- so it still will never return `False`.